packages/apps/app-dashboard/src/utils/user-dashboard/lit.ts

```diff
@@ -166,7 +167,17 @@ export async function registerWebAuthn(displayName: string): Promise<IRelayPKP>
     !userResponse.pkpPublicKey ||
     !userResponse.pkpEthAddress
   ) {
-    throw new Error('Minting failed: Invalid response data');
+    const error = new Error('Minting failed: Invalid response data');
+    Sentry.captureException(error, {
+      extra: {
+        context: 'lit.registerWebAuthn',
+        responseStatus: userResponse.status,
+        hasPkpTokenId: !!userResponse.pkpTokenId,
+        hasPkpPublicKey: !!userResponse.pkpPublicKey,
+        hasPkpEthAddress: !!userResponse.pkpEthAddress,
+      },
+    });
+    throw error;
```

Comment:
Almost same as above, duplicating the capture. On this case we can [add breadcrumbs](https://docs.sentry.io/platforms/javascript/enriching-events/breadcrumbs/) to not lose details but leave the error capture/resolution to the one that receives it afterwards